boards/arc/emsdp/emsdp_em4.dts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019, Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <emsdp.dtsi>
+#include "board.dtsi"
+
+/ {
+	model = "emsdp";
+	compatible = "snps,emsdp", "snps,emsdp";
+
+	aliases {
+		uart-0 = &uart0;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+};
+
+/ {
+	soc {
+
+		uart@f0004000 {
+			interrupts = <108 1>;
```

Comment:
Is this really a board level change?  Feels like this should possible be in a soc dtsi file?

Reply:
emsdp board is similar to em_starterkit board, it is FPGA based board which supports different core configurations, such as EM4,
EM5D, EM6, EM7D, EM9D, EM9D+ESP, EM11D. in different core configurations the interrupts number may be different. so we need to point it out in core configuration related dts file.

Reply:
Since these are different core/SoC I think this should move into dts/arc/em{5d,6,7,9d,9d+esp,11d}.dtsi

Reply:
Currently, most of differences of  different core are board-specific. Not easy to move into dts/arc/

We know there are lots of duplications,  plan to do a cleanup with other arc boards together in future.